packages/gatsby/src/utils/api-browser-docs.js

```diff
@@ -206,6 +206,10 @@ exports.onServiceWorkerInstalled = true
  * Inform plugins of when a service worker has an update available.
  * @param {object} $0
  * @param {object} $0.serviceWorker The service worker instance.
+ * @example
+ * exports.onServiceWorkerUpdateFound = () => {
+ *   window.alert(`An updated service worker was found.`)
```

Comment:
We generally want to keep these examples relatively useful (e.g. users will probably directly copy and paste this).

Is this the best example we could show? I'm almost inclined to suggest reverting it back to the way it was, but minimally I think perhaps a prompt like you've shown earlier is more effective?